### PR TITLE
fixing Bringup and DCM_bringup launch files

### PR DESCRIPTION
**Pull request by nlyubova**
Let's fix the Bringup launch files (there is a bug with the sensor launch),
And exclude the torso and the head from the DCM_bringup launch files, unfortunately it still makes the robot shaking

### CODE REVIEW

**Comment by vrabaud** [comment]
is the .xacro supposed to be part of that PR ?

**Comment by nlyubova** [comment]
oops, I've removed it

**Comment by nlyubova** [comment]
please wait a moment before merging, there is one more issue

**Comment by nlyubova** [comment]
I've added the data from cameras (top and bottom) and they are published well, but there is some issue with topic names. Could you please, accept it in this state at the moment?

File: romeo_sensors_py/nodes/camera.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+import rospy
+from pepper_sensors.pepper_camera import PepperCam
```
Comment (vrabaud) [inline romeo_sensors_py/nodes/camera.py]: Pepper ? So what's the deal with romeo_camera.py ? Why is there a PepperCam in there too ?

File: romeo_sensors_py/setup.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```
Comment (vrabaud) [inline romeo_sensors_py/setup.py]: If you have Python, you need to update your CMakeLists.txt:
http://docs.ros.org/jade/api/catkin/html/user_guide/setup_dot_py.html

You should not install scripts through setup.py a mentioned on that page too.

**Comment by nlyubova** [comment]
Changed!

File: romeo_sensors_py/setup.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+    scripts=['nodes/camera.py', 'nodes/camera_depth.py'],
```
Comment (vrabaud) [inline romeo_sensors_py/setup.py]: remove scripts from here.

File: romeo_dcm/romeo_dcm_control/launch/romeo_dcm_position_control.launch [hunk 4]
```diff
@@ -12,8 +12,10 @@
      /romeo_dcm/LHipYawPitch_position_controller /romeo_dcm/LHipRoll_position_controller /romeo_dcm/LHipPitch_position_controller /romeo_dcm/LKneePitch_position_controller /romeo_dcm/LAnklePitch_position_controller /romeo_dcm/LAnkleRoll_position_controller
      /romeo_dcm/RHipYawPitch_position_controller /romeo_dcm/RHipRoll_position_controller /romeo_dcm/RHipPitch_position_controller /romeo_dcm/RKneePitch_position_controller /romeo_dcm/RAnklePitch_position_controller /romeo_dcm/RAnkleRoll_position_controller
      /romeo_dcm/joint_state_controller
-     /romeo_dcm/NeckYaw_position_controller /romeo_dcm/NeckPitch_position_controller /romeo_dcm/HeadPitch_position_controller /romeo_dcm/HeadRoll_position_controller
-     /romeo_dcm/TrunkYaw_position_controller
      "/>
      
+    <!--
```
Comment (vrabaud) [inline romeo_dcm/romeo_dcm_control/launch/romeo_dcm_position_control.launch]: don't move lines around: you can comment inside <node
It would make things easier to track.

File: romeo_dcm/romeo_dcm_driver/src/romeo.cpp [hunk 5]
```diff
@@ -60,7 +60,7 @@ bool Romeo::initialize()
                             "LWristYaw",
                             "LWristPitch",
                             "LHand",
-                            "TrunkYaw",
+                            /*"TrunkYaw",
```
Comment (vrabaud) [inline romeo_dcm/romeo_dcm_driver/src/romeo.cpp]: so you disable the legs too ?

Comment (nlyubova) [inline romeo_dcm/romeo_dcm_driver/src/romeo.cpp]: Yes unfortunately, only arms can be used without shaking...

File: romeo_sensors_py/package.xml [hunk 6]
```diff
@@ -11,12 +11,8 @@
   <author email="hris2003@gmail.com">Ha Dang</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-
-  <exec_depend>cv_bridge</exec_depend>
-  <exec_depend>naoqi_sensors_py</exec_depend>
-  <exec_depend>rospy</exec_depend>
-
-  <export>
```
Comment (vrabaud) [inline romeo_sensors_py/package.xml]: this package is still architecture independent, please keep that. It is needed on Fedora.

File: romeo_sensors_py/package.xml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package format="2">
+<package>
```
Comment (vrabaud) [inline romeo_sensors_py/package.xml]: why removing ?

File: romeo_sensors_py/package.xml [hunk 8]
```diff
@@ -11,12 +11,8 @@
   <author email="hris2003@gmail.com">Ha Dang</author>
 
   <buildtool_depend>catkin</buildtool_depend>
-
-  <exec_depend>cv_bridge</exec_depend>
```
Comment (vrabaud) [inline romeo_sensors_py/package.xml]: you need cv_bridge in camera_depth.py . Why did you remove that ?

File: romeo_sensors_py/setup.py (hunk 2 above)
Comment (nlyubova) [inline romeo_sensors_py/setup.py]: thanks!

File: romeo_sensors_py/package.xml (hunk 6 above)
Comment (nlyubova) [inline romeo_sensors_py/package.xml]: ok, got it back

File: romeo_sensors_py/package.xml (hunk 7 above)
Comment (nlyubova) [inline romeo_sensors_py/package.xml]: now, it's back

File: romeo_sensors_py/package.xml (hunk 8 above)
Comment (nlyubova) [inline romeo_sensors_py/package.xml]: that was a mistake, sorry

File: romeo_sensors_py/setup.py (hunk 3 above)
Comment (nlyubova) [inline romeo_sensors_py/setup.py]: done

File: romeo_dcm/romeo_dcm_control/launch/romeo_dcm_position_control.launch (hunk 4 above)
Comment (nlyubova) [inline romeo_dcm/romeo_dcm_control/launch/romeo_dcm_position_control.launch]: done!

**Comment by nlyubova** [comment]
Now, it is better, and still working :)

File: romeo_sensors_py/package.xml [hunk 9]
```diff
@@ -12,11 +12,11 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <exec_depend>cv_bridge</exec_depend>
+  <exec_depend>cv_bridge</exec_depend>
```
Comment (vrabaud) [inline romeo_sensors_py/package.xml]: space at the end : revert the file simply

File: romeo_sensors_py/package.xml [hunk 10]
```diff
@@ -12,11 +12,11 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <exec_depend>cv_bridge</exec_depend>
+  <exec_depend>cv_bridge</exec_depend> 
   <exec_depend>naoqi_sensors_py</exec_depend>
   <exec_depend>rospy</exec_depend>
 
-  <export>
+  <export>
```
Comment (vrabaud) [inline romeo_sensors_py/package.xml]: space at the end : revert the file simply

Comment (nlyubova) [inline romeo_sensors_py/package.xml]: done

File: romeo_sensors_py/package.xml (hunk 9 above)
Comment (nlyubova) [inline romeo_sensors_py/package.xml]: sorry

**Comment by nlyubova** [comment]
@vrabaud could you please accept this forgotten pull request?

**Comment by vrabaud** [comment]
@suryaambrose, please look at those repos too.
